plugins/exporter.koplugin/main.lua

```diff
@@ -183,7 +183,11 @@ function Exporter:exportClippings(clippings)
                     v.timestamp = timestamp
                     local status = v:export(exportables)
                     if status then
-                        table.insert(statuses, _(v.name .. ": Exported to " ) .. v:getFilePath(exportables))
+                        if v.is_remote then
+                            table.insert(statuses, _(v.name .. ": Exported successfully."))
```

Comment:
Urgh, I didn't realize there was an _() around these; I guess I thought they were just some debugging messages. If those need to be translated, it should be either:

```lua
v.name .. ": " .. _("Exported successfully.")
```

Or

```
T(_("%1: Exported successfully."), v.name)
```

Could you add a screenshot to show what the actual result looks like?